src/validator.cpp

```diff
@@ -35,6 +35,38 @@ limitations under the License.
 
 namespace libcellml {
 
+/**
+* @brief Validate that equivalent variable pairs in the @p model
+* have equivalent units.
+* Any errors will be logged in the @c Validator.
+*
+* Any difference in base units is reported as an error in the @c Validator, but the multiplier difference does not trigger a validator error.
+* Where the base units are equivalent, the multiplier may be interpreted as units_of_v1 = (10^multiplier)*units_of_v2
```

Comment:
I think we should change this to be consistent with other multipliers - ie: return the actual multiplier, not the log10 of it?

Reply:
And also the order - in the Units file it's units_of_v2 = units_of_v1*multiplier ?

Reply:
On second thoughts, if this isn't shown to users then I'm happy enough